### PR TITLE
feat/table plugin menu

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-table/web/src/tiptap/decorations/pluginMenuDecoration.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Decoration } from 'prosemirror-view';
+import styles from '../../../statics/styles/side-toolbar.scss';
+import { PLUGIN_MENU_HORIZONTAL_MODAL_ID } from 'ricos-types';
+import type { Layout, Placement } from 'wix-rich-content-common';
+import { pluginMenuIcon } from './svgs';
+
+export const getPluginMenuDecoration = services => $cell => {
```
Comment (danarish4) [inline packages/plugin-table/web/src/tiptap/decorations/pluginMenuDecoration.ts:7]: Do you have a reason for returning a callback?

Comment (talevy17) [reply packages/plugin-table/web/src/tiptap/decorations/pluginMenuDecoration.ts:7]: nope, shouldn't be a callback (:

File: packages/plugin-table/web/src/tiptap/plugins/columnResizingPlugin.tsx [hunk 2]
```diff
@@ -85,10 +85,11 @@ function handleMouseMove(view, event, key) {
   }
 }
 
-function handleMouseDown(view, event, key) {
+function handleMouseDown(view, event, editor, key) {
   const pluginState = key.getState(view.state);
+  const { editCell: isCellEditingActive } = editor.state['cell-states$'] || {};
 
-  if (pluginState.activeHandle === -1 || pluginState.dragging) return false;
+  if (pluginState.activeHandle === -1 || pluginState.dragging || isCellEditingActive) return false;
```
Comment (danarish4) [inline packages/plugin-table/web/src/tiptap/plugins/columnResizingPlugin.tsx:92]:
```suggestion
  if (pluginState.activeHandle === -1 || pluginState.dragging || event.target.contains('pluginMenuButton')) return false;
```

**Review by danarish4 — CHANGES_REQUESTED** [review]
Please check if the click is on plugin add button instead of disable resizing plugin.

Bonus points 😄 :
Enable column resizing only from column handler

File: packages/plugin-table/web/src/tiptap/plugins/cellStatesPlugin.tsx [hunk 3]
```diff
@@ -57,7 +57,8 @@ export const cellStatesPlugin = (Plugin, PluginKey, editor) => {
         const tablePluginState = state && key.getState(state);
         if (tablePluginState.editCell) {
           const editDecorations = cellEditingDecorations(tablePluginState.editCell);
-          return DecorationSet.create(state.doc, editDecorations);
+          const pluginMenuDecoration = getPluginMenuDecoration(services)(tablePluginState.editCell);
```
[the remark this answers is not in the document]
Comment (talevy17) [reply packages/plugin-table/web/src/tiptap/plugins/cellStatesPlugin.tsx]: already changed its outdated

File: packages/plugin-table/web/src/tiptap/plugins/columnResizingPlugin.tsx (hunk 2 above)
Comment (talevy17) [reply packages/plugin-table/web/src/tiptap/plugins/columnResizingPlugin.tsx]: redundant (:

**Pull request merged**